Bloggist/src/aura/progression/progressionController.js

```diff
@@ -0,0 +1,40 @@
+({
+	itemsChange: function(component, event) {
+       var status = component.get("v.statusValue");
+       var liDraft = component.find("draftLi");
+       var liReady = component.find("readyLi");
+       var liPublish = component.find("publishLi");
+        if(status === 'Draft'){
+          $A.util.removeClass(liDraft, 'slds-is-incomplete');
+          $A.util.addClass(liDraft, 'slds-is-current');
```

Comment:
You should actually implement all this in the markup using conditional classes based on which value statusValue has.